examples/asr/asr_vad/README.md

```diff
@@ -0,0 +1,40 @@
+# NeMo ASR+VAD Inference
```

Comment:
Let's add this feature to the main readme of nemo and also add it to the docs.

Reply:
Done~

Reply:
Has it been added to the docs?

Reply:
Not added to docs yet, haven't decided where to add it. Let's do it later, since we'll have a buffered inference version later and we can update the docs together